dist.ini

```diff
@@ -1,31 +1,12 @@
 name    = Email-Stuffer-TestLinks
-author  = binary.com <BINARY@cpan.org>
+author  = bianry.com <BINARY@cpan.org>
```

Comment:
```suggestion
author  = binary.com <BINARY@cpan.org>
```

Reply:
Wow. Did I make such mistake for all PRs ? Thanks let me check